tests/unit/test_runtime.py

```diff
@@ -90,10 +105,143 @@ def test_runtime_grants(mocker):
         sql_backend = MockBackend()
         crawl_grants(cfg, ws, sql_backend, mock_installation())
 
-        assert "SHOW DATABASES FROM hive_metastore" in sql_backend.queries
+        assert "SELECT * FROM hive_metastore.ucx.grants" in sql_backend.queries
+
+
+# TODO crawl_permissions fails, but I don't think I can create tickets yet!
```

Comment:
it seems the changes with commented code from the other PR leaked to this one. 

let's remove the non-rename code from this PR.

Reply:
Yes the reason for this was to ensure that it fixed the problem encountered in the other PR.
I'll rebase.

Reply:
(That said if you approve the other PR the resulting change should be equivalent)

Reply:
let's remove the commented-out code :) it's always bad.